File: src/main/java/com/getlose/Ticket.java
package com.getlose;

public class Ticket {

    int origin;
    int price;
    int destination;

    public Ticket() {
    }

    public Ticket(int origin, int price, int destination) {
        this.origin = origin;
        this.price = price;
        this.destination = destination;
    }

    public int getOrigin() {
        return origin;
    }

    public void setOrigin(int origin) {
        this.origin = origin;
    }

    public int getPrice() {
        return price;
    }

    public void setPrice(int price) {
        this.price = price;
    }

    public int getDestination() {
        return destination;
    }

    public void setDestination(int destination) {
        this.destination = destination;
    }
}

File: src/main/kotlin/com/getlose/Ticket.kt
package com.getlose

import java.util.*

data class Ticket(val origi:Int, val destination: Int, val price:Int)

fun main(args: Array<String>) {
    var ticket = Ticket(1,22,20)
    val s = "33444433"
    print(s.validate())
    //println((1..10).random())
    println(Random().nextInt(10)+1)
}

fun String.validate() : Boolean{
    return this.length >= 6
}

fun IntRange.random():Int{
    return Random().nextInt(endInclusive-start)+start
}